java/code/src/com/suse/manager/webui/websocket/Notification.java

```diff
@@ -162,7 +163,7 @@ public void onError(Session session, Throwable err) {
      * @param message the message to be sent
      */
     public static void sendMessage(Session session, String message) {
-        synchronized (session) {
+        synchronized (LOCK) {
```

Comment:
with this change one can only send a message at the same time, in an application, instead of per session. Is this waht you want to archive?

Reply:
yeah, this fixes a SonarQube error about using a method variable as lock and aligns it with the other functions using LOCKs in that code